core/kernel/Float_spec.rb

```diff
@@ -198,6 +198,53 @@ def to_f() 1.2 end
     end
   end
 
```

Comment:
Could we wrap all the new specs in
`describe "for hexadecimal representations"` or some better description?